### ISSUE TITLE
Error when applying

### ISSUE THREAD

**Issue by thokari** [michel-kraemer/gradle-download-task#100]
Hi, I suddenly get this weird error from this simple build file:

```
plugins {
    id 'de.undercouch.download' version '3.3.0'
}
```

```
$ ./gradlew
```

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'test'.
> Could not resolve all files for configuration ':classpath'.
   > Could not find de.undercouch:gradle-download-task:3.3.0.
     Searched in the following locations:
         https://plugins.gradle.org/m2/de/undercouch/gradle-download-task/3.3.0/gradle-download-task-3.3.0.pom
         https://plugins.gradle.org/m2/de/undercouch/gradle-download-task/3.3.0/gradle-download-task-3.3.0.jar
     Required by:
         project :

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
```

Really weird isn't it? Any idea?

**Comment by felipecsl** [comment]
weird, im seeing the same issue. sounds like a problem with plugins.gradle.org

**Comment by timyates** [comment]
Hi all!

It's a [problem with versions of the `plugin-publish` plugin prior to version `0.9.7`](https://discuss.gradle.org/t/plugin-authors-please-use-the-latest-plugin-publish-plugin-others-may-result-in-a-broken-upload/23573)

It causes the portal to think everything went well, but in actual fact the artifacts have not been stored correctly.

@michel-kraemer, I can remove v3.3.0 from the portal, and you can re-publish once the `plugin-publish` plugin version is updated?

**Comment by DTU-STR** [comment]
Have the same problem:
https://plugins.gradle.org/m2/de/undercouch/gradle-download-task/3.3.0/gradle-download-task-3.3.0.jar
=> 303 -> https://plugins-artifacts.gradle.org/de.undercouch/gradle-download-task/3.3.0/38376f178f96c7526fa054cc5171adbc0edba84e8fc6a33cc1093b5328703675/gradle-download-task-3.3.0.jar
=> 404 message:
<Message>The specified key does not exist.</Message>
<Key>
de.undercouch/gradle-download-task/3.3.0/38376f178f96c7526fa054cc5171adbc0edba84e8fc6a33cc1093b5328703675/gradle-download-task-3.3.0.jar
</Key>

Please republish.

**Comment by michel-kraemer** [comment]
Thanks! This has been fixed by @timyates in 57553fe5538a3a3a444662a96d611262fa327371. I just re-uploaded version 3.3.0 to the plugin portal. Sorry for the inconvenience.

Please try again and re-open the issue if the problem persists.

**Comment by timyates** [comment]
:+1:
